01-arch/07-reputation.md

```diff
@@ -0,0 +1,41 @@
+## Reputation system
+
+\Gls{NeoFS} reputation system is a subsystem for calculating trust in \Gls{Node}. It is based on a reputation model for assessing trust, which is, in turn, based on the EigenTrust algorithm designed for peer-to-peer reputation management. The algorithm ensures that there is a uniquely defined manager (parent node) for each network participant at each specified time (epoch). Based on the information received from its child node and other managers, it iteratively puts a complex general (Global) Trust of the entire network into the applicable child node.
+
+### Trust
+
+Trust in a \Gls{NeoFS} \Gls{Node} is its quantitative (numerical) assessment based on the experience of interacting with that \Gls{Node}. The higher the score is, the higher is the trust in the node and vice versa. Since the system uses a reputation-based trust model, the terms “trust” and “reputation” are considered synonymous in this documentation.
+
+*The Subject of trust assessment*
+: is the one who calculates trust.
+
+*The Object of trust assessment*
+: is the one whose trust is being calculated.
+
+Reputation models are based on the nature of social media reputation. Thus, trust in \Gls{Node} is built up both by the estimates of the behavior of the \Gls{Node} by another \Gls{Node} and by the reputation of the \Gls{Node} evaluating its behavior.
+
+Trust in a \Gls{NodeFS} in \Gls{Node} is formed based on its interactions with the Subjects of trust assessment. Therefore, the reputation of a \Gls{Node} changes during the \Gls{NeoFS} network working cycle when both the behavior of the node itself, and the behavior of other nodes change.
+
+### Algorithm
+
+*General problem statement:* the Subject of assessment needs to calculate the reputation of the Object of trust assessment at a specific point of time (specific epoch).
+
+*Local Trust*
+: trust of one \Gls{Node} to another, calculated using *only* statistical information of their peer-to-peer network interactions. The Subject and Object of such trust are peer-to-peer nodes.
+
+*Global Trust*
+: the result of the [EigenTrust](http://ilpubs.stanford.edu:8090/562/1/2002-56.pdf) algorithm -- the trust in the network participant, which was obtained regarding *all* Local Trusts of *all* nodes. The Subject is the *entire* network as a whole, the Object is a separate node.
```

Comment:
I suggest to add EigenTrust section into the Glossary, the same way it's done for HRW